### DISCUSSION TITLE
❌  Root Layout: Failed to execute 'removeChild' on 'Node' error!

### DISCUSSION THREAD

**Discussion by lithika-damnod** [vercel/next.js#54346] [App Router]
**I created a layout.js file inside the "(Auth)" directory, but it conflicts with the main Root Layout,** 

<img width="296" alt="image" src="https://github.com/vercel/next.js/assets/61654812/68b6322d-8d7d-48b9-a790-7c21a4cb9795">
<br><br>

Code for _Main Root Layout_  👇
```
'use client'

import './globals.css'
import { Inter } from 'next/font/google'

import Navigation from '@/components/ui/Navigation'
import store from './store'
import { Provider } from 'react-redux'

const inter = Inter({ subsets: ['latin'] })

export default function RootLayout({ children }) {
  return (
    <Provider store={store}>
      <html lang="en">
        <body className={inter.className}>
          <Navigation />

          {children}
        </body>
      </html>
    </Provider >
  )
}

```

Code for _(Auth)/layout.js_  👇
```import { Inter } from 'next/font/google'
const inter = Inter({ subsets: ['latin'] })

export default function RootLayout({ children }) {
    return (
        <html lang="en">
            <body className={inter.className}>
                {children}
            </body>
        </html>
    )
}
```
<br><br>
> When I visit the route "/login", I want the navigation bar to be removed, but it isn't 🤕

<img width="1437" alt="image" src="https://github.com/vercel/next.js/assets/61654812/e56659a2-4e2c-47e8-add0-3658b7eca111">
<br><br>

> Also, when I hover over the navigation bar
> I get the error ❌

<img width="1354" alt="image" src="https://github.com/vercel/next.js/assets/61654812/7a0a6d33-3c4b-4408-80d1-769521583850">

**Comment by yaffalhakim1** [comment]
hey, are you already find the answer?
